docs/developer/bigquery/bigquery_support.md

```diff
@@ -0,0 +1,25 @@
+# BigQuery support for EDC
```

Comment:
could you please follow the decision record template? You can grasp it from other [decision records](https://github.com/eclipse-edc/Connector/tree/main/docs/developer/decision-records), also the folder structure should be the same

Reply:
i think this serves more as a technical description, a decision record (which should go into the DR folder structure) will likely follow later.

Reply:
This was the basis for the discussion, I didn't mean to add the decision log yet, before exchanging with you.

Reply:
Updating to DR template